.env-sample

```diff
@@ -1,18 +0,0 @@
-APP_NAME=Wikonnect
```

Comment:
Do not delete the contents of this file. It is essential for anyone wishing to setup the project using Docker.